src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanel.cs

```diff
@@ -48,27 +59,38 @@ public AttachPanel(EvasObject conformant)
         /// <param name="conformant">The caller's conformant.</param>
         /// <exception cref="OutOfMemoryException">Thrown when an attempt to allocate the memory fails.</exception>
         /// <exception cref="InvalidOperationException">Thrown when the AttachPanel already exists or the <paramref name="conformant"/> is not a conformant object.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the AttachPanel is not supported in the device.</exception>
+        /// <exception cref="ArgumentNullException">Thrown when the parameter is null</exception>
         public AttachPanel(Conformant conformant)
```

Comment:
It is redundancy code, because Conformant is EvasObject

```c#
        public AttachPanel(Conformant conformant) : this(conformant as EvasObject)
        {
        }
```
It is enough

Reply:
Yes totally I agree with that. done.